minlzlib/minlzlib.h

```diff
@@ -72,6 +72,7 @@ void RcSetDefaultProbability(uint16_t* Probability);
 //
 bool LzDecode(void);
 bool LzInitialize(uint8_t Properties);
+extern const uint8_t LzSupportedProperties;
```

Comment:
If you check out https://github.com/ionescu007/minlzma/pull/4 you'll see that some people like to use minlzlib as 'amalgamated source files' (I think GCC/clang even have a flag to do this automatically). As per that issue, it seems that extern const breaks that, so what I did to solve that PR was to introduce RcSetDefaultProbability. I would recommend doing something similar here, and introduce LzGetDefaultProperties.

Reply:
Since I was adding a new exposed function anyway, I moved everything in `LzInitialize` after the properties value check to a new separate function called `LzResetState()` (which `LzInitialize` now calls), and in `lzma2dec` I now just call `LzResetState` directly.